test/DynamoCoreTests/Configuration/PreferenceSettingsTests.cs

```diff
@@ -375,7 +375,7 @@ PreferencesComparison comparePrefenceSettings(PreferenceSettings defaultSettings
         }
 
         [Test]
-        [Category("UnitTests"), Category("FailureNET6")]
+        [Category("UnitTests")]
```

Comment:
unfortunately this is passing locally, but fails on CI - I will investigate further...
```
  The file D:\jenkins\workspace\Dynamo\DynamoSelfServe\pullRequestValidation\Dynamo\test\settings\DynamoSettings-NewSettings.xml exist: True | DiffProps: 1 | TotProps: 60 | Default Sett NumberFormat: f3 | New Sett NumberFormat: f3 | First Property with the same value IsFirstRun
  Expected: True
  But was:  False
```